lib/cucumber/runtime.rb

```diff
@@ -166,11 +166,13 @@ def set_encoding
 
     require 'cucumber/formatter/ignore_missing_messages'
     require 'cucumber/formatter/fail_fast'
+    require 'cucumber/formatter/publish_banner_printer'
     require 'cucumber/core/report/summary'
     def report
```

Comment:
could we pop a newline above this to separate require concerns

Reply:
done on master